citation-server/src/gitasdb/write.js

```diff
@@ -10,6 +10,12 @@ import { readObject } from './read';
 
 const logger = winston.loggers.get('GitAsDb');
 
+export function emptyField(field) {
```

Comment:
This function returns true if the field is considered empty and false otherwise.
To be considered empty, a field must be an empty string or an object with an empty array or object as value